packages/ckeditor5-link/tests/linkui.js

```diff
@@ -2436,11 +2436,15 @@ describe( 'LinkUI', () => {
 				) ).to.be.true;
 
 				expect( getModelData( editor.model ) ).to.equal(
-					'<paragraph>fo<$text linkHref="http://ckeditor.com">http://ckeditor.com</$text>[]ar</paragraph>'
+					'<paragraph>fo' +
+						'<$text linkHref="http://ckeditor.com">http://</$text>' +
+						'<$text bold="true" linkHref="http://ckeditor.com">ckeditor.com</$text>' +
+						'<$text bold="true">[]</$text>ar' +
```

Comment:
The bold attribute is set on the selection after updating a link. Can we check how it behaves on the current master? I'm not sure if the selection should not have overridden gravity here. But looks like a case for a follow-up.

Reply:
On the master, there is no `'<$text bold="true">[]</$text>ar'`. I thought it's feature, as the bold attribute will be inherited directly start typing after applying command.